ref(angular): Update Angular SDK docs for `@sentry/angular-ivy` (#6356)
getsentry/sentry-docs: src/wizard/javascript/angular.md

type: framework
---

> To use Sentry with your Angular application, you will need to use `@sentry/angular-ivy` or `@sentry/angular`, Sentry’s Browser Angular SDKs:
> 

Reviewer: ```suggestion
To use Sentry with your Angular application, you'll need `@sentry/angular-ivy` or `@sentry/angular`, Sentry’s Browser Angular SDKs:

```

- If you're using Angular 12 or newer, use `@sentry/angular-ivy`
- If you're using Angular 10 or 11, use `@sentry/angular`

Add the Sentry SDK as a dependency using `yarn` or `npm`:

```bash
# Using yarn (Angular 12+)
yarn add @sentry/angular-ivy @sentry/tracing
# Using yarn (Angular 10 and 11)
yarn add @sentry/angular @sentry/tracing

# Using npm (Angular 12+)
npm install --save @sentry/angular-ivy @sentry/tracing
# Using npm (Angular 10 and 11)

> npm install --save @sentry/angular @sentry/tracing

Author: I know this is ugly but afaik we can't use tabs in the onboarding wizard :(
